src/panels/config/cloud/alexa/cloud-alexa.ts

```diff
@@ -170,17 +175,38 @@ class CloudAlexa extends LitElement {
                 secondary-line
                 @click=${this._showMoreInfo}
               >
-                ${entity.entity_id in this.hass.entities
-                  ? html`<button
-                      class="link"
-                      .entityId=${entity.entity_id}
-                      @click=${this._openAliasesSettings}
-                    >
-                      ${this.hass.localize(
-                        "ui.panel.config.cloud.alexa.manage_aliases"
-                      )}
-                    </button>`
-                  : ""}
+                ${aliases
+                  ? html`
+                      <button
+                        class="link"
+                        .entityId=${entity.entity_id}
+                        @click=${this._openAliasesSettings}
+                        aria-label=${this.hass.localize(
```

Comment:
The button has text, so it should not have an `aria-label`.  By doing this, you are telling the user agent to ignore the child text.  Seems like this text ought to be consumed by everyone anyway as it actually describes the button's action.